### PR TITLE
adding italia/republic-history/

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
Please include this new bunch of negotiations pertaining to The "Portal of Sources for the History of the Italian Republic" project.

### CODE REVIEW

**Comment by davidlehn** [comment]
You're adding rules under `/italia/`, this needs approval from one of the maintainers of that space:
@bfabio @mfortini @giorgialodi

**Review by bfabio — APPROVED** [review]

**Comment by bfabio** [comment]
Thanks @davidlehn, lgtm.

**Pull request merged**
